code/modules/cargo/bounties/assistant.dm

```diff
@@ -0,0 +1,238 @@
+	reward = 1200
+	wanted_types = list(/obj/item/relic)
+
+/datum/bounty/item/assistant/scooter
+	name = "Scooter"
+	description = "Nanotrasen has determined walking to be wasteful. Ship a scooter to CentCom to speed operations up."
+	reward = 1080 // the mat hoffman
+	wanted_types = list(/obj/vehicle/ridden/scooter)
+	include_subtypes = FALSE
+
+/datum/bounty/item/assistant/skateboard
+	name = "Skateboard"
+	description = "Nanotrasen has determined walking to be a wasteful. Ship a skateboard to CentCom to speed operations up."
+	reward = 900 // the tony hawk
+	wanted_types = list(/obj/vehicle/ridden/scooter/skateboard)
+*/
+/datum/bounty/item/assistant/stunprod
+	name = "Stunprod"
+	description = "CentCom demands a stunprod to use against dissidents. Craft one, then ship it."
+	reward = 1300
+	wanted_types = list(/obj/item/weapon/melee/baton/cattleprod)
+
+/datum/bounty/item/assistant/soap
+	name = "Soap"
+	description = "Soap has gone missing from CentCom's bathrooms and nobody knows who took it. Replace it and be the hero CentCom needs."
+	reward = 2000
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/soap)
+
+/datum/bounty/item/assistant/spear
+	name = "Spears"
+	description = "CentCom's security forces are going through budget cuts. You will be paid if you ship a set of spears."
+	reward = 2000
+	required_count = 5
+	wanted_types = list(/obj/item/weapon/material/twohanded/spear)
+
+/datum/bounty/item/assistant/toolbox
+	name = "Toolboxes"
+	description = "There's an absence of robustness at Central Command. Hurry up and ship some toolboxes as a solution."
+	reward = 2000
+	required_count = 6
+	wanted_types = list(/obj/item/weapon/storage/toolbox)
+
+/datum/bounty/item/assistant/statue
+	name = "Statue"
+	description = "Central Command would like to commision an artsy statue for the lobby. Ship one out, when possible."
+	reward = 2000
+	wanted_types = list(/obj/structure/sculpting_block)
+
+/datum/bounty/item/assistant/statue/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/structure/sculpting_block/S = O
+	if(S && S.sculpted)
+		return TRUE
+	return FALSE
+
+/datum/bounty/item/assistant/cheesiehonkers
+	name = "Cheesie Honkers"
+	description = "Apparently the company that makes Cheesie Honkers is going out of business soon. CentCom wants to stock up before it happens!"
+	reward = 1200
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/cheesiehonkers)
+
+/datum/bounty/item/assistant/baseball_bat
+	name = "Baseball Bat"
+	description = "Baseball fever is going on at CentCom! Be a dear and ship them some baseball bats, so that management can live out their childhood dream."
+	reward = 2000
+	required_count = 5
+	wanted_types = list(/obj/item/weapon/material/twohanded/baseballbat)
+
+/datum/bounty/item/assistant/donut
+	name = "Donuts"
+	description = "CentCom's security forces are facing heavy losses against the Syndicate. Ship donuts to raise morale."
```

Comment:
This is probably going to be solved at the end, but you should be careful of references that would make sense on /tg/ but not on AuroraStation. A lot of these are a little memey; you should bother the lore writers to come up with excuses to get these items.